src/components/_overlays/MemberDropdown/MemberDropdown.styles.ts

```diff
@@ -0,0 +1,113 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+import { Avatar } from '@/components/Avatar'
+import { Text } from '@/components/Text'
+import { cVar, media, sizes, transitions, zIndex } from '@/styles'
+
+const paddingStyles = css`
+  padding: ${sizes(6)} ${sizes(4)};
+`
+
+export const Container = styled.div<{ isActive: boolean }>`
+  position: fixed;
+  right: ${sizes(4)};
+  top: 0;
+  width: 280px;
+  max-height: calc(${window.innerHeight - sizes(4, true)}px - var(--size-topbar-height));
```

Comment:
Couldn't we use just `100vh` instead? We have a guarantee this will always be responsive, `window.innerHeight` is iffy

Reply:
yeah this is better just pushed it